src/words/TokTok.jl

```diff
@@ -101,51 +101,56 @@ function toktok_tokenize(instring::AbstractString)
         atoms(ts, rules_atoms) ||
         character(ts)
     end
-    flush!(ts)
+    if flush_later1 == nothing
+        flush!(ts)
+    elseif flush_later2 == nothing
+        flush!(ts, flush_later1)
+    else
+        flush!(ts, flush_later1, flush_later2)
+    end
+
     return ts.tokens
 end
 
 """
     handle_final_periods(::TokenBuffer)
 Handles the following rules from original toktok perl script:
 Don't tokenize period unless it ends the line and that it isn't preceded by another period (FINAL_PERIOD_1)
-Don't tokenize period unless it ends the line(FINAL_PERIOD_2)
+Don't tokenize period unless it ends the line (FINAL_PERIOD_2)
 """
 function handle_final_periods(ts::TokenBuffer)
     effective_end = length(ts.input)
     # handles FINAL_PERIOD_1 = r"(?<!\.)\.$"
     if length(ts.input) >= 2 && ts.input[end] == '.' && ts.input[end-1] != '.'
-        flush!(ts, ".")
         effective_end -= 1
-        return effective_end
+        return effective_end, ".", nothing
     end
 
     # handles FINAL_PERIOD_2 = r"(?<!\.)\.\s*(["'’»›”]) *$"
-    if ts.input[end] in "“”‘’›" || isspace(ts.input[end])
```

Comment:
Shouldn't this also have `\"` in it based on regex?

Reply:
Yes